piker/data/marketstore.py

```diff
@@ -132,7 +132,10 @@ def start_marketstore(
 
     mktsdir = os.path.join(config._config_dir, 'marketstore')
 
-    # create when dne
+    # create dirs when dne
```

Comment:
Wait, won't this fail since you're trying to make the `/marketstore` subdir *afterward*?

I'd luv to see a test verifying this creation sequence 😉

Reply:
Ah wait nm, estoy pendejo, i see the order is correct..